src/components/subscriptions/constants.js

```diff
@@ -21,3 +21,11 @@ export const DEFAULT_PAGE = 1;
 
 // used to determine whether to show the revocation cap messaging in the license revoke modal
 export const SHOW_REVOCATION_CAP_PERCENT = 80;
+
+// Subscription expiration
+// Days until expiration threshold constants
+export const SUBSCRIPTION_EXPIRATION_FIRST_THRESHOLD = 120;
```

Comment:
I don't love these names I came up with

Reply:
`SUBSCRIPTION_DAYS_REMAINING_MODERATE`, ``SUBSCRIPTION_DAYS_REMAINING_SEVERE`, `SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL`.

https://droughtmonitor.unl.edu/About/AbouttheData/DroughtClassification.aspx

Reply:
or maybe just `export const SUBSCRIPTIONS_REMAINING_DAYS_THRESHOLDS = [30, 60, 120]`?

Reply:
Love the drought classification. Perfection